README.md

```diff
@@ -585,11 +585,12 @@ Finds one document matching a `filter` where:
     - `err` - if the query failed, the error reason, otherwise `null`.
     - `result` - if the query succeeded, a document as a class instance.
 
-#### `findOneAndUpdate(filter, [options], callback)`
+#### `findOneAndUpdate(filter, update, [options], callback)`
 
 Finds one document matching a `filter`, updates it and returns it where:
 
  - `filter` - a filter object used to select the document to update.
+ - `update` - The update document.
```

Comment:
Let's use the same working as `findByIdAndUpdate`

`update` - an object containing the fields/values to be updated.